praw/models/reddit/live.py

```diff
@@ -37,3 +59,96 @@ def __init__(self, reddit, id=None,  # pylint: disable=redefined-builtin
 
     def _info_path(self):
         return API_PATH['liveabout'].format(id=self.id)
+
+    @property
+    def fullname(self):
+        """Return ``name`` of :class:`.LiveThread`."""
+        return 'LiveUpdateEvent_' + self.id
+
+    def updates(self, **generator_kwargs):
+        """Return a :class:`.ListingGenerator` yields :class:`.LiveUpdate` s.
+
+        :param generator_kwargs: keyword arguments passed to
+            :class:`.ListingGenerator` constructor.
+        :returns: A :class:`.ListingGenerator` object which yields
+            :class:`.LiveUpdate` object.
+        """
+        path = API_PATH['live_updates'].format(id=self.id)
+        for update in ListingGenerator(self._reddit, path,
+                                       **generator_kwargs):
+            update.thread_id = self.id
```

Comment:
Instead of setting `thread_id` could we set a `_thread` attribute which links to the actual thread, and have a `thread` property which returns that value? The reason I prefer this approach is that it can be used to convey the value is not returned from the API, but rather added by PRAW.